### PR TITLE
Use geocodeio

### PR DESCRIPTION
**Pull request by iandees**
A first stab at switching over to geocode.io to do district lookup.

### CODE REVIEW

**Comment by iandees** [comment]
This isn't ready to go yet… I haven't tested it yet and I'm not sure what the input or output data is supposed to look like.

File: index.html [hunk 1]
```diff
@@ -94,29 +94,17 @@
...
-                    break;
-                } else if (key.match(/ocd-division\/country:us\/state:../)) {
-                    var splitKey = key.split('/');
-                    stateName = splitKey[2].substring(6).toUpperCase();
-                    districtNumString = '0';
-                }
-            }
+            let jsonResults = res.results;
+            var districtNumString = jsonResults[0].fields.congressional_districts[0].district_number;
+            var stateName = jsonResults[0].address_componenets.state;
```
Comment (jillh510) [inline index.html]: is it really spelled componenets?

**Review by jillh510 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (iandees) [reply index.html]: Hah, nope.

**Comment by iandees** [comment]
This is ready to go, @jillh510. Tested it locally and it works well for me, including the case of not finding a matching zipcode.

**Review by jillh510 — APPROVED** [review]

**Pull request merged**